source/api/commands/invoke.md

```diff
@@ -2,7 +2,7 @@
 title: invoke
 ---
 
-Invoke a function on the previously yielded subject.
+Invoke a function on the previously yielded subject. If the invoked function returns a promise, it will automatically wait for it until it resolves.
```

Comment:
I wouldn't include this in the first line of the description since it's a detail of how it works in a very specific circumstance. This type of detail about 'wait' and 'retry' usually goes within the #Rules section here: https://docs.cypress.io/api/commands/invoke.html#Rules